tests/test_clock.py

```diff
@@ -30,3 +32,57 @@ def test_negative_numbers_raise_ValueError(self):
     def test_now_is_timezone_aware(self):
         """now() should always return a tz-aware datetime."""
         self.assertIsNotNone(self.clock.now().tzinfo)
+
+
+class TimerTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_clock = mock.Mock()
+
+    def test_timer_duration_exceeded(self):
+        duration = datetime.timedelta(hours=3 * 24)
+        self.mock_clock.now.side_effect = [
+            datetime.datetime(
+                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
+            datetime.datetime(
+                2016, 7, 27, 10, 51, 9, 928000, tzinfo=pytz.utc)
+        ]
+        self.assertTrue(clock.Timer(self.mock_clock, duration).expired())
```

Comment:
(minor) Consider using rounder numbers to make it easier to see at a glance what the difference is between the two dates like

```python
            datetime.datetime(
                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc),
            datetime.datetime(
                2000, 1, 4, 0, 0, 0, 0, tzinfo=pytz.utc)
```